config/overlays/openshift/manager_run_flags_patch.yaml

```diff
@@ -0,0 +1,16 @@
+# This patch inject a sidecar container which is a HTTP proxy for the
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+spec:
+  template:
+    spec:
+      containers:
+        - command: ["go run"]
```

Comment:
Why `go run` and not the binary directly? As per go binaries being selfsustained we don't need to load the golang runtime into target containers and thus squeeze them in total size.